scripts/utils/migrate/get-frontmatter.js

```diff
@@ -99,6 +99,8 @@ const addCustomFrontmatter = {
     return {
       ...defaultFrontmatter,
       summary: doc.summary || '',
+      docsID: doc.docId,
```

Comment:
```suggestion
      id: doc.docId,
```

Could we call this `id` instead? In the context of the document, this feels a bit nicer.